src/command_modules/azure-cli-servicefabric/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.0.7
++++++
+* Added details errors to validation response when creating cluster.
```

Comment:
Move this to be under the 0.0.6 heading then remove the 0.0.7 heading

Reply:
Typo 'details' -> 'detailed'?

Reply:
Done, sry for the typo